### PR TITLE
Rewrite [GithubTag]

### PR DESCRIPTION
**Pull request by paulmelnikow**
Ref #2863

### CODE REVIEW

**Comment by calebcartwright** [comment]
I added a unit test for `render` and have approved 👍 (will defer to someone else to merge since I've added a commit)

File: services/github/github-tag.spec.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      'v1.3-beta3'
+    )
+    given({ json: tagFixture, usingSemver: false, includePre: false }).expect(
+      'cheese'
+    )
+  })
+
+  test(GithubTag.render, () => {
+    given({ usingSemver: false, version: '1.2.3' }).expect({
+      message: addv('1.2.3'),
```
Comment (paulmelnikow) [inline services/github/github-tag.spec.js]: Any reason not to replace these with literals?

Comment (calebcartwright) [reply services/github/github-tag.spec.js]: I've no issues with replacing them. Rationale very much related to https://github.com/badges/shields/pull/3266#discussion_r278374394, the latter paragraph in particular

Comment (calebcartwright) [reply services/github/github-tag.spec.js]: `message`'s inlined, still calling the `versionColor` helper below but happy to change that to `blue` if you prefer (and re 👍 the PR)

Comment (paulmelnikow) [reply services/github/github-tag.spec.js]: I'm happy to continue the discussion at https://github.com/badges/shields/pull/3266#discussion_r278374394 though that's a bit more long-ranging; we can revisit this. For now I'd prefer we inline the color as well.

**Pull request merged**